Library/Homebrew/dev-cmd/prof.rb

```diff
@@ -1,12 +1,28 @@
 #:  * `prof` [<ruby options>]:
 #:    Run Homebrew with the Ruby profiler.
-#:    For example:
-#:      brew prof readall
+#:
+#:    *Example:* `brew prof readall`
+
+require "cli_parser"
```

Comment:
Cannot use `CLI::Parser` in `prof.rb` as of now. Thats because all the options have to be declared and `prof` takes in ruby options which we cannot declare. Need to add a functionaly in `CLI::Parser` to be able to ignore unknown options.

Reply:
Does the same issue affect `test.rb`?

Reply:
No this issue does not affect `test.rb` as it accepts only specific set of options described in the documentation, so we can use `CLI::Parser` there.

Reply:
`brew prof` doesn't have to take arbitrary arguments; we can document the `brew` we want to accept and expand them if necessary?

Reply:
(like `brew ls`)